test/e2e/specs/editor/various/editor-modes.spec.js

```diff
@@ -155,4 +155,35 @@ test.describe( 'Editing modes (visual/HTML)', () => {
 			},
 		] );
 	} );
+
+	test( 'should reparse changes from code editor', async ( {
+		editor,
+		page,
+		pageUtils,
+	} ) => {
+		// Open the code editor.
+		await pageUtils.pressKeys( 'secondary+M' );
+
+		const textbox = page.getByRole( 'textbox', {
+			name: 'Type text or HTML',
+		} );
+
+		// Change content by typing.
+		await textbox.fill( '' );
+		await textbox.focus();
+
+		await page.keyboard.type( `<!-- wp:paragraph -->
+<p>abc</p>
+<!-- /wp:paragraph -->` );
```

Comment:
@ellatrix, is typing action here intentional or can we just use `fill`?

I'm asking because typing produces block validation warnings and creates a lot of noise in e2e test reports, which can hide the real issues.

Reply:
It should be fine if the test still passes

Reply:
Created a follow-up https://github.com/WordPress/gutenberg/pull/62957.